source/default/_patterns/00-protons/tailwind.tokens.css

```diff
@@ -14,10 +14,81 @@ Example: Google Remote Fonts
 Serif - Google Fonts - Oswald
 Sans-Serif - Google Fonts - Open Sans
 Monospace - Google Fonts - Source Code Pro
-*/
+*/ 
 
 @import url(https://fonts.googleapis.com/css?family=OpenSans|Oswald|SourceCodePro&display=swap);
 
+
+/* Button Variations */
+@layer components {
+  .btn {
+    @apply text-center align-middle select-none mb-4 font-semibold py-1.5 px-3 text-base;
+  }
+  .btn--navy,
+  .btn-primary,
+  .btn-dark {
+    @apply bg-utc-new-blue-500 text-white 
+    border-2 border-utc-new-blue-500 hover:bg-blue-900 hover:text-white focus:shadow-inner;
+  }
+
+  .btn--yellow,
+  .btn-danger,
+  .btn-secondary
+   {
+    @apply bg-utc-new-gold-500 text-utc-new-blue-500 border-2 border-utc-new-gold-500 hover:bg-utc-new-gold-600 hover:text-utc-new-blue-500 hover:border-utc-new-gold-600 focus:shadow-inner;
+  }
+
+  .btn--navy-outline,
+  .btn-info {
+    @apply bg-white text-utc-new-blue-500 border-2 border-utc-new-blue-500 hover:bg-utc-new-blue-500 hover:text-white hover:border-utc-new-blue-500 hover:shadow-inner focus:shadow-inner;
+  }
+
+  .btn--lightblue,
+  .btn-light {
+    @apply bg-gray-300 text-utc-new-blue-500 border-2 border-gray-300 hover:bg-gray-400 hover:text-utc-new-blue-500 hover:border-gray-400 focus:shadow-inner;
+  }
+
+  .btn--burgundy,
+  .btn-warning {
+    @apply bg-red-900 text-white border-2 border-red-900 hover:bg-utc-new-gold-500 hover:text-red-900 hover:border-utc-new-gold-500 focus:shadow-inner;
+  }
+
+  .btn--white-outline {
+   @apply bg-none text-white border-2 border-white hover:bg-utc-new-gold-500 hover:text-utc-new-blue-500 hover:border-white focus:shadow-inner;
+  }
+
+  .btn--green,
+  .btn-success {
+    @apply bg-green-800 text-white border-2 border-green-800 hover:bg-green-700
+    hover:text-white hover:border-green-700 focus:shadow-inner;
+  }
+
+  .btn--small {
+    @apply py-1 px-2 text-sm
+  }
+
+  .btn--medium {
+    @apply py-1.5 px-3 text-base;
+  }
+
+  .btn--large {
+   @apply py-2 px-4 text-lg;
+  }
+
+  .btn--extra-large {
+    @apply p-6 text-xl font-bold font-utcheadings;
+   }
```

Comment:
so font-utcheadings; did not work?

Reply:
No, it didn't apply Exo when I used font-utcheadings (which I saw defined in the font-family file). I tried to figure out why but couldn't crack it.

Reply:
https://tailwindcss.com/docs/extracting-components

Reply:
It may be the order in which they were place maybe? 
`.btn--extra-large {
  @apply font-bold font-utcheadings p-6 text-xl ;
 }` 
I got it working with the text above as seen in the image below.
![Screen Shot 2021-08-30 at 2 24 29 PM](https://user-images.githubusercontent.com/39039024/131386701-86767c19-c646-42b7-8b74-c8487bedcfaa.png)

Reply:
@brittrich

Reply:
I had the same font application problem in working with the PL demo for the cards. It wasn't picking up the load so I manually placed it in the twig file as an additional import. I will look at Bernardo's solution.

Reply:
Hmm. So, I tried reordering and it still didn't work. I added the Exo import to the tailwind tokens css file and that seems to have kicked it in everywhere...here and the headings as well. Think that's the right spot for it? I saw Open Sans import but no Exo so figured it was worth a try to add it.